### DISCUSSION TITLE
Favicon transparency of docs.github.com

### DISCUSSION THREAD

**Discussion by jordr** [community/community#114422] [Other Feature Feedback, Questions, & Ideas]
Forwarding from Support ticket.

Description

https://github.com/ uses a transparent favicon. This is nice, works on non-white backgrounds

https://docs.github.com/ uses a favicon with no transparency that looks similar to the github.com favicon on a white background, but terrible on other backgrounds

That's it!

Best,
@jordr

**Comment by Nel-S** [comment]
Bumping this. I was originally going to file a report for the same issue (as admittedly completely minor as it is).

**Comment by fater** [comment]
**It looks like different teams are doing it.**
I saw the same case in our company - we were creating similar products on subdomains based on the main project, and so late noticed differences styles ))

Eventually, when the teams notice this, they will make the same transparent icons everywhere. ))
